tests/example_workflows/pyxaid2/example_1_Si/step3/run_step3.py

```diff
@@ -129,8 +129,23 @@
 for i in xrange(len(params["Phi_basis"])):
     params["Phi_dE"].append(0.0)
 
+### Initialize paramters to scale energies and NACs ###
+en_gap = 3.21 # units in eV
+params["Chi_dE"] = []
+params["NAC_dE"] = []
+# For each Chi state, appead the energy correction between state Chi[n] and Chi[n+1]
+params["Chi_dE"].append( en_gap )
```

Comment:
for consistency, I suggest that all the input we provide is in atomic units, so the user would do the conversion in the script themselves